sympy/solvers/tests/test_inequalities.py

```diff
@@ -323,15 +337,16 @@ def test_issue_10198():
         -1 + 1/abs(1/x - 1) < 0) == Or(
         And(-oo < x, x < 0), And(S(0) < x, x < S(1)/2)
         )
-    raises(NotImplementedError, lambda: reduce_inequalities(abs(1/sqrt(x)) - 1, x))
+    assert reduce_inequalities(abs(1/sqrt(x)) - 1, x) == Or(
+        And(-oo < x, x < 0), Eq(x, 1))
     assert reduce_abs_inequality(-3 + 1/abs(1 - 1/x), '<', x) == \
         Or(And(-oo < x, x < 0),
         And(S(0) < x, x < S(3)/4), And(S(3)/2 < x, x < oo))
     raises(ValueError,lambda: reduce_abs_inequality(-3 + 1/abs(
         1 - 1/sqrt(x)), '<', x))
 
 def test_issue_10047():
-    assert solve(sin(x) < 2) == And(-oo < x, x < oo)
+    assert solve(sin(x) < 2) == And(S(0) <= x, x < 2*pi)
```

Comment:
> It's generally not a good idea to change unit tests, especially when these are here for regression purposes for a specific solved issue.

Yes. 
In this scenario, we have changed the solving technique.
We have restricted the domain of the solution to a periodic interval.
I changed the test because I thought that it would be better to display this behaviour throughout the test suite.

Should I XFAIL this or modifying the test is better ?
Ping @moorepants @hargup @aktech